internal/services/healthcare/healthcare_dicom_resource.go

```diff
@@ -30,9 +30,9 @@ func resourceHealthcareApisDicomService() *pluginsdk.Resource {
 		Delete: resourceHealthcareApisDicomServiceDelete,
 
 		Timeouts: &pluginsdk.ResourceTimeout{
-			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Create: pluginsdk.DefaultTimeout(00 * time.Minute),
```

Comment:
`0` minutes is going to be problematic, presumably this is meant to be `90`?

Reply:
Thanks Tom, will do